db/db_test_util.cc

```diff
@@ -671,19 +671,13 @@ std::string DBTestBase::Get(int cf, const std::string& k,
   return result;
 }
 
-std::string DBTestBase::Get(const std::string& k, PinnableSlice* v) {
+Status DBTestBase::Get(const std::string& k, PinnableSlice* v,
+                       std::string& value) {
   ReadOptions options;
   options.verify_checksums = true;
-  std::string result;
   Status s = dbfull()->Get(options, dbfull()->DefaultColumnFamily(), k, v);
-  if (s.IsNotFound()) {
-    result = "NOT_FOUND";
-  } else if (!s.ok()) {
-    result = s.ToString();
-  }  else {
-    result = std::string(v->data());
-  }
-  return result;
+  value = std::string(v->data());
```

Comment:
What is that you need to convert PinnableSlice to string? Seems redundant, does not it? Can't the function that calls ::Get read the value from v?